tests/unit/stimulus/text_test.py

```diff
@@ -0,0 +1,163 @@
+# Copyright (c) 2024 The pymovements Project Authors
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+from pathlib import Path
+
+import pytest
+
+import pymovements as pm
+
+
+@pytest.mark.parametrize(
+    ('aoi_file'),
+    [
+        pytest.param(
+            'tests/files/toy_text_1_1_aoi.csv',
+            id='toy_text_1_1_aoi',
+        ),
+        pytest.param(
+            'tests/files/toy_text_2_2_aoi.csv',
+            id='toy_text_2_2_aoi',
+        ),
+        pytest.param(
+            'tests/files/toy_text_3_3_aoi.csv',
+            id='toy_text_3_3_aoi',
+        ),
+    ],
+)
+def test_str_aoi_path(aoi_file):
+    pm.stimulus.text.from_file(
```

Comment:
these tests currently do not check for expected results.

there are a few things you can check (in separate test cases or combined, as you wish).

- is the returned object a `TextStimulus`
- does the `TextStimulus.aois` dataframe have the resulting columns that are expected from the example file?
- does the `TextStimulus.aois` dataframe have the expected number of rows expected from the example file?

Reply:
done